gst/nnstreamer/tensor_transform/tensor_transform.c

```diff
@@ -689,20 +689,41 @@ gst_tensor_transform_set_option_data (GstTensorTransform * filter)
     }
     case GTT_STAND:
     {
-      gchar **strv = NULL;
-
-      strv = g_strsplit (filter->option, ":", -1);
+      gchar **options = NULL;
+      guint i, num_options;
 
-      filter->data_stand.mode = gst_tensor_transform_get_stand_mode (strv[0]);
       filter->data_stand.out_type = _NNS_END;
-      if (g_strv_length (strv) > 1)
-        filter->data_stand.out_type = gst_tensor_get_type (strv[1]);
+      filter->data_stand.per_channel = FALSE;
+
+      options = g_strsplit (filter->option, ",", -1);
+      num_options = g_strv_length (options);
+
+      for (i = 0; i < num_options; i++) {
+        gchar **strv = g_strsplit (options[i], ":", -1);
+
+        if (g_ascii_strcasecmp (strv[0], "default") == 0 ||
```

Comment:
Q: I wonder `g_strv_length (strv) > 0` is always guaranteed.

Reply:
IMO if num_options is larger than 0, options[i] is not null, so this is guaranteed.

Reply:
I missed one thing, now no regex to match stand options, but I will update new PR to add regex for stand mode :)

Reply:
Yes, thanks for the explanation.